auth-web/src/components/auth/create-account/AccountCreateBasic.vue

```diff
@@ -151,7 +150,7 @@ export default class AccountCreateBasic extends Mixins(Steppable) {
   }
 
   private get enablePaymentMethodSelectorStep (): boolean {
-    return LaunchDarklyService.getFlag(LDFlags.PaymentTypeAccountCreation) || false
+    return true
```

Comment:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/8358481b-d87d-4026-aee6-f7fd99e45d4d)

Reply:
Same as above. This one is a tiny bit trickier.

Reply:
Thanks. Looks good.